code/game/objects/structures/displaycase.dm

```diff
@@ -301,6 +301,8 @@
 		return
 	if(user.combat_mode)
 		return ..()
+	if(W.tool_behaviour == TOOL_WELDER && !user.combat_mode && !broken)
```

Comment:
wtf my welder collection will be ruined now noooooooooooooooooooooooooooooo

also, you don't need to check for !combat_mode here, because combat_mode is checked for in the check above this one

Reply:
also, are you still unable to repair trophy display cases with glass?

Reply:
> also, are you still unable to repair trophy display cases with glass?

I didn't change any methods of repair for display cases, just this weird snowflake case.

Reply:
somewhat related to this- do trophy display cases still require you to be on combat mode to repair them with glass?